src/chrome/content/rules/Mozilla.net.xml

```diff
@@ -1,45 +1,124 @@
-		- hardhat ²
-	² Works; mismatched, CN: gp1.wac.edgecastcdn.net
+	Time out:
+		dynamicua-origin.cdn.mozilla.net
+		start-origin.cdn.mozilla.net
+		start-origin-phx1.cdn.mozilla.net
+		videos-real-origin.cdn.mozilla.net
+		download-speed-test.mozilla.net
+		addons.lb1.phx.mozilla.net
+		addons-stage.lb1.phx.mozilla.net
+		snippets.lb1.phx.mozilla.net
+		mozilla-org.lb1.phx.mozilla.net
+		pp-zlb04.phx.mozilla.net
+		pp-zlb07.phx.mozilla.net
+		addons.zlb.phx.mozilla.net
+		pyrepo.addons.zlb.phx.mozilla.net
+		addons-dev.zlb.phx.mozilla.net
+		addons-dev-nginx.zlb.phx.mozilla.net
+		addons-services-single1.zlb.phx.mozilla.net
+		addons-services-single2.zlb.phx.mozilla.net
+		addons-single.zlb.phx.mozilla.net
+		addons-single1.zlb.phx.mozilla.net
+		addons-single2.zlb.phx.mozilla.net
+		addons-stage.zlb.phx.mozilla.net
+		addons-versioncheck.lb1.phx.mozilla.net
+		addons-versioncheck-bg.lb1.phx.mozilla.net
+		addons-versioncheck-bg.zlb.phx.mozilla.net
+		addons-versioncheck-single1.zlb.phx.mozilla.net
+		addons-versioncheck-single2.zlb.phx.mozilla.net
+		addons-versioncheck-single3.zlb.phx.mozilla.net
+		amo.zlb.phx.mozilla.net
+		bouncer01.zlb.phx.mozilla.net
+		bugzilla.zlb.phx.mozilla.net
+		bugzilla-wild.zlb.phx.mozilla.net
+		confluence.zlb.phx.mozilla.net
+		crash-stats.zlb.phx.mozilla.net
+		crash-stats-prod.zlb.phx.mozilla.net
+		crash-stats-stage.zlb.phx.mozilla.net
+		download.zlb.phx.mozilla.net
+		download-mirror02.zlb.phx.mozilla.net
+		download-stats.zlb.phx.mozilla.net
+		generic-dev.zlb.phx.mozilla.net
+		generic-non-ssl.zlb.phx.mozilla.net
+		generic-stage.zlb.phx.mozilla.net
+		intra.zlb.phx.mozilla.net
+		intranet.zlb.phx.mozilla.net
+		keys.zlb.phx.mozilla.net
+		lb1.phx1.mozilla.net
+		ldap.zlb.phx.mozilla.net
+		mozillians-python.zlb.phx.mozilla.net
+		mozorg-engagement.zlb.phx.mozilla.net
+		pfs.zlb.phx.mozilla.net
+		phonebook.zlb.phx.mozilla.net
+		puppetdashboard.zlb.phx.mozilla.net
+		samo.zlb.phx.mozilla.net
+		snippets.zlb.phx.mozilla.net
+		snippets-stats.zlb.phx.mozilla.net
+		static-redirect.zlb.phx.mozilla.net
+		startpage.zlb.phx.mozilla.net
+		static.zlb.phx.mozilla.net
+		svn.zlb.phx.mozilla.net
+		vamo.zlb.phx.mozilla.net
+		webowonderorg-engagement.zlb.phx.mozilla.net
+		wild-add-ons.zlb.phx.mozilla.net
+		www-mozilla-org.zlb.phx.mozilla.net
+		www-mozilla-org-multi.zlb.phx.mozilla.net
+		lb1a.phx1.mozilla.net
+		lb1b.phx1.mozilla.net
+		mktns.services.phx1.mozilla.net
+		dataviz.zlb.scl3.mozilla.net
+		addons-mozilla-org.mktns.services.phx1.mozilla.net
+		addons-mozilla-org-san.mktns.services.phx1.mozilla.net
+		addons-redirects.mktns.services.phx1.mozilla.net
+		allizom.mktns.services.phx1.mozilla.net
+		getpersonas.mktns.services.phx1.mozilla.net
+		static-addons-mozilla-net.mktns.services.phx1.mozilla.net
+		nat-solitude.svc.phx1.mozilla.net
+		vmimages.mozilla.net
+		conserver1.untrust.yvr1.mozilla.net
+		lo0.border1.untrust.yvr1.mozilla.net
+		lo0.border2.untrust.yvr1.mozilla.net
```

Comment:
You can ignore all hosts that do not support HTTP/HTTPS

Reply:
@J0WI I also ignore the ones that only support HTTPS.